app/models/game.rb

```diff
@@ -1,10 +1,35 @@
 class Game < ActiveRecord::Base
 
-  has_many :memberships
-  has_many :users, through: :memberships
+  validates :lobby, presence: true, uniqueness: true
 
-  def add_user(user)
-    memberships.create user: user
+  belongs_to :lobby
+
+  before_create :setup_game
+
+  CARD_ATTRIBUTES = {
+    symbol:  [ :diamond, :sguiggle, :oval ],
+    number:  [ :one, :two, :three ],
+    color:   [ :red, :green, :purple ],
+    shading: [ :solid, :striped, :open ]
+  }
+
+  def deck
```

Comment:
I agree. That functionality belongs in `GameSerializer`, since it's role is to display a game (which this is all about anyway). I'll make the change in the other feature branch to avoid having to wrangle with fixing it here only to have major portions of `dealer` and `game` change.